inception-project/inception
# Feature/849 add a description field to kb profiles

**What's in the PR**
* Adding a KnowledgeBaseInfo class to KnowledgeBaseProfiles to show more detailed information about the pre-configured knowledge bases in the wizard.

**How to test manually**
* Create a knowledge base in the wizard. Check that info-fields/labels are filled correctly
* Feel free to suggest layout changes
* Correct information for UKP Wikidata 

**Automatic testing**
* [X] PR includes unit tests (changes tests)

**Documentation**
